runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -43,6 +43,7 @@
 #include "env/VMAccessCriticalSection.hpp"      // for VMAccessCriticalSection
 #include "runtime/RuntimeAssumptions.hpp"
 #include "env/J9JitMemory.hpp"
+#include "optimizer/FearPointAnalysis.hpp"
```

Comment:
really not a fan of this - see below we need a more sensible header to hold the common query if there isn't already a suitable OSR header.